_posts/2016-10-18-gymkhanalive.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Gymkhana L!VE"
+date: 2016-10-18 16:42:00 +0530
+author: 11000011
```

Comment:
Also, Please don't close this pull request. Just make your changes and then:
```
$ git add .
```
followed by
```
git commit --amend
```
after that do
```
git push -f origin master
```